project/plugins.sbt

```diff
@@ -0,0 +1,3 @@
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC13")
+addSbtPlugin("ch.epfl.scala" % "sbt-release-early" % "2.0.0")
+addSbtPlugin("com.swoval" %% "sbt-mac-watch-service" % "1.1.4")
```

Comment:
I wouldn't add this plugin here, it should be installed globally per machine imo

Reply:
Okay